_data/events/London-Pivotal.json

```diff
@@ -0,0 +1,15 @@
+{
+  "title": "GDCR 17 @ Pivotal London",
+  "url": "https://www.eventbrite.com/e/global-day-of-code-retreat-pivotal-tickets-37119002937",
+  "moderators": [
+    "@deniseyu21",
+		"jamesjoshuahill",
+ 		"chewymeister"
+  ],
```

Comment:
Can you please add `@` to the handles so we can recognize them as twitter handles?